02/app.js

```diff
@@ -9,12 +9,16 @@ class App extends React.Component {
         usersList: [],
     }
 
+    updateList = newUser => {
+        this.setState(({ usersList }) => { return { usersList: [...usersList, newUser] } })
```

Comment:
Jeśli w jednej linii to może bez `return`: `this.setState(({ usersList }) => ({ usersList: [...usersList, newUser] }))` (dodałem nawias okrągły.